README.md

```diff
@@ -1,58 +1,88 @@
+2) Download the latest JAR from the releases tab.
+3) When you install QuPath, folder for extensions is set */home/user/QuPath/extensions* by default 
+(in Linux). You can put the JAR in this folder or just drag-and-drop it in the QuPath window.
+> For new version another QuPath user directory is recommended to avoid conflict with the previous 
+> extension version. You can set another user directory. Press edit -> preferences and 
+> set another user directory in QuPath. Also, you can delete the previous extension version, if 
+> you don't want to use another QuPath user directory.
 
-## Setting up project to use dicom cloud:
-To upload to the cloud it is necessary to dicomize your whole-slide images using embedded dicomizer in qupath-cloud extension or 
-using [OrthancWSIDicomizer](https://www.orthanc-server.com/browse.php?path=/whole-slide-imaging) and upload it with the help of [cURL](https://curl.haxx.se/)  or [dcm4che stowrs](https://sourceforge.net/projects/dcm4che/).
+## Setting up project to use DICOM cloud:
 
 1) Create empty project in QuPath.
-2) Click **Cloud** button on the button bar. Login with OAuth in your default browser with your account which have access permission (1 time).
-3) Select project and click **NEXT** button in extension window.
-4) Create dataset. Click **New Dataset** button, input dataset name, choose near location and after that click **NEXT** button.
-5) Create dicom store. Click **New DICOM Store** button, input dicom store name and after that click **OK** button.
-6) Upload dicomized images into created dicom store and dataset in chosen location:
-   1) If you want to use the embedded dicomizer in the qupath-cloud extension, just add images to the project. When you click **Synchronize** button images will be dicomized and uploaded.
-   2) Alternatively you can use [OrthancWSIDicomizer](http://book.orthanc-server.com/plugins/wsi.html). 
-   Once source images have been dicomized, you can upload them via [curl](https://cloud.google.com/healthcare/docs/how-tos/dicom-import-export) or [dcm4che stowrs tool](https://github.com/dcm4che/dcm4che/tree/master/dcm4che-tool/dcm4che-tool-stowrs).
-7) Click **Synchronize** button on the button bar in QuPath to synchronize DICOM Store with your project.
->Extension saved in your chosen dicom store for existing project and saved your access permission on computer.   
->Supported formats for qupath-cloud extension and pathology test data you can see [here](https://openslide.org/).
+2) Click the **Cloud** button on the button bar. Login with OAuth in your default browser with your 
+account which has access permission to Cloud Healthcare API (1 time).
+3) Select the desired Project and click the **NEXT** button in extension window.
+4) Select an existing Dataset or create a new one. Steps to create a new Dataset:
+   1) Click the **New Dataset** button.
+   2) Input the Dataset name.
+   3) Choose the desired Location in where you want the Dataset stored.
```

Comment:
"Select the location where the Dataset should be stored"?

Reply:
Done